pkg/apis/resmgr/v1alpha1/types.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019-2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	resapi "k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	resmgr "github.com/intel/cri-resource-manager/pkg/apis/resmgr"
+)
+
+const (
+	GroupName string = "criresmgr.intel.com"    // GroupName is the group of our CRD.
```

Comment:
I wanted to use `cri-resource-manager.intel.com` here and in the schema. But unfortunately the code generator can't handle that gracefully. It generates syntactically invalid golang code including variable names with dashes ('-').

A bit of googling around left me with the impression that this is known bug, so/and it will be eventually fixed... IMO, we can switch to a final namespace then if we still want to. I don't know if anything else would be a better intermediate namespace.

Reply:
Ok. I think `criresmgr` is good enough at this stage.